.github/workflows/ci.yml

```diff
@@ -49,8 +50,18 @@ jobs:
              -Plicense-check \
             | tee ${{ env.MVN_BUILD_OUTPUT_FILE }}
 
+          mvn ${MVN_COMMON_OPTIONS} dependency:tree -Plicense-check | tee ${{ env.MVN_DEPENDENCY_PLUGIN_OUTPUT_FILE }}
+
       - name: Check licensing
         run: |
-          mvn ${MVN_COMMON_OPTIONS} exec:java@check-licensing -N \
```

Comment:
align with naming in pom

Reply:
That might deserve it's own hotfix commit

Reply:
yep, i was thinking about that however had some doubts... 
Since you've also mentioned this I think we should do that

Reply:
extracted into a separate commit